recipes.js

```diff
@@ -1,11 +1,65 @@
-const mongoose = require('mongoose');
-const Schema   = mongoose.Schema;
-const data = require('./data.js')
+const chalk = require("chalk");
```

Comment:
You don't need the chalk package, here :) This package was just to train you to install npm packages, but we are actually never going to use this one !